test/packages/parallel/apache/_dev/build/docs/README.md

```diff
@@ -1,34 +1,129 @@
-# Apache Integration
+{{ `{{header}}` }}
+# Apache Integration for Elastic
 
-This integration periodically fetches metrics from [Apache](https://httpd.apache.org/) servers. It can parse access and error
-logs created by the Apache server.
+## Overview
 
-## Compatibility
+The Apache integration for Elastic enables collection of access logs, error logs and metrics from [Apache](https://httpd.apache.org/) servers.
+This integration facilitates performance monitoring, understanding traffic patterns and gaining security insights from
+your Apache servers.
 
-The Apache datasets were tested with Apache 2.4.12 and 2.4.46 and are expected to work with
-all versions >= 2.2.31 and >= 2.4.16 (independent from operating system).
+### Compatibility
 
-## Logs
+This integration is compatible with all Apache server versions >= 2.4.16 and >= 2.2.31.
 
-### Access Logs
+### How it works
 
-Access logs collects the Apache access logs.
+Access and error logs data stream read from a logfile. You will configure Apache server to write log files to a location that is readable by elastic-agent, and 
+elastic-agent will monitor and ingest events written to these files.
 
-{{fields "access"}}
```

Comment:
It looks like `fields` are not rendered anymore. In the rendered README I can only see `{{fields}}`, in https://github.com/elastic/elastic-package/pull/2707/files#diff-21c266b51e67f0cc833e6b48a09867796a652d1b2d9abe69119b3f36d7975d99R114.